packages/shared/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@ls1intum/apollon": "2.15.0",
+    "@ls1intum/apollon": "2.15.2",
```

Comment:
@matthiaslehnertum isn't the [latest version](https://www.npmjs.com/package/@ls1intum/apollon?activeTab=versions) `2.15.1`?

Reply:
Now, version 1.5.2 is released with the styling fix for the scale controls - sorry, I thought I would be able to push this release sooner. 😅